apps/music/test/marionette/Player_test.js

```diff
@@ -267,6 +267,37 @@ marionette('Music player tests', function() {
     });
   });
 
+  suite('Repeat test', function() {
+    test('Check that repeat button works', function() {
+      try {
+        music.launch();
+        music.waitForFirstTile();
+        music.switchToSongsView();
```

Comment:
Here you should also have a ````music.waitForListEnumerate()```` to make sure the list is loaded before you even try to look for the first item.